home/graphql/queries/ProfileDataQuery.query.graphql

```diff
@@ -0,0 +1,34 @@
+query Home_ProfileData2($appLimit: Int!, $snackLimit: Int!) {
```

Comment:
Looking around - seems like we could get rid of the 2 here (and in the hook's name)

Reply:
I'm hesitant to change these since they're unique-ish and used to identify this query in datadog, which is useful for seeing usage of Expo Go. The hook is autogenerated from this.
 
That being said I'm not opposed to changing it, but either way we should do it in a separate PR to limit the scope of changes of this one in case a revert is ever needed of one or the other.